treemap.swt/src/main/java/de/engehausen/treemap/swt/TreeMap.java

```diff
@@ -238,7 +256,11 @@ protected synchronized void recalculate() {
 				buildControl.cancel();
 				buildControl = null;
 			}
-			setCursor(getDisplay().getSystemCursor(SWT.CURSOR_WAIT));
+
+			if ( changeCursorOnRecalculate ) {

```

Comment:
the whitespaces around the parentheses should be removed, see above

Reply:
Done.